src/storage-types/src/dyncfgs.rs

```diff
@@ -22,7 +22,19 @@ pub const DELAY_SOURCES_PAST_REHYDRATION: Config<bool> = Config::new(
         (namely, upsert) till after rehydration is finished",
 );
 
+/// Whether rendering should use `mz_join_core` rather than DD's `JoinCore::join_core`.
```

Comment:
nit: copy-pasta on the rustdoc (and also the "This was original `false`" comment?)

Reply:
🤦, will clean this up!